### ISSUE TITLE
Diaphora produces error while adding manual match

### ISSUE THREAD

**Issue by r0ny123** [joxeankoret/diaphora#268]
The following error occurs:
```
[Diaphora: Sat Aug 12 16:15:25 2023] Adding manual match between string_decryption_routine and sub_401058
Traceback (most recent call last):
  File "C:/Users/Rony/Desktop/Tools/diaphora\diaphora_ida.py", line 305, in activate
    return self.obj.OnCommand(sel, self.cmd_id)
  File "C:/Users/Rony/Desktop/Tools/diaphora\diaphora_ida.py", line 573, in OnCommand
    self.add_manual_match()
  File "C:/Users/Rony/Desktop/Tools/diaphora\diaphora_ida.py", line 645, in add_manual_match
    self.add_manual_match_internal(name1, name2)
  File "C:/Users/Rony/Desktop/Tools/diaphora\diaphora_ida.py", line 602, in add_manual_match_internal
    ratio = self.bindiff.compare_function_rows(main_row, diff_row)
  File "C:\Users/Rony/Desktop/Tools/diaphora\diaphora.py", line 2566, in compare_function_rows
    main_d["ea"] = main_row["address"]
TypeError: 'NoneType' object is not subscriptable
```

**Comment by joxeankoret** [comment]
I can repro, thanks!

**Comment by joxeankoret** [comment]
Should be fixed with 706308b. Feel free to reopen if it does not.
